xcode/Subconscious/Shared/Components/FirstRun/FirstRunProfileView.swift

```diff
@@ -13,18 +13,34 @@ struct FirstRunProfileView: View {
     @Environment(\.colorScheme) var colorScheme
     
     var body: some View {
+        let did = Did(app.state.sphereIdentity ?? "") ?? Config.default.subconsciousGeistDid
         NavigationStack {
-            VStack {
+            VStack(spacing: AppTheme.padding) {
```

Comment:
<img src="https://github.com/subconsciousnetwork/subconscious/assets/58421/ecfc2221-41e0-429c-b118-e571bef721d8" width="500"/>

Reply:
I pared this view back to the essentials since I moved the pfp to the next step. I also centered everything and hid the button based on keyboard focus.

<img width="377" alt="Screenshot 2023-05-11 at 11 22 50 am" src="https://github.com/subconsciousnetwork/subconscious/assets/5009316/24ee8e4a-6359-4f6e-99ab-befe3453d762">